utils/errors.go

```diff
@@ -0,0 +1,44 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"net/url"
+	"reflect"
+)
+
+var UnexpectedErrorMsg = "unexpected error"
+
+// HandleBasicApiError Handles API Error response which contains a payload with a message
+// Returns an unexpected error when this doesn't happen
+func HandleBasicApiError(err error) error {
+	urlErr, isUrlErr := err.(*url.Error)
+	if isUrlErr {
+		return fmt.Errorf(
+			"'%s' not found, please verify the provided server URL or check your internet connection",
+			urlErr.Err.(*net.OpError).Err.(*net.DNSError).Name,
```

Comment:
I think this should be changed in #45, to something like `Config.ServerURL`

Reply:
Yes, we should, also because those type assertions can panic if the error type is not the expected one!

Reply:
Yes, I think it's actually better to just read the server URL from the env var so I'll change it